examples/stable-diffusion/text_to_image_generation.py

```diff
@@ -241,10 +243,10 @@ def main():
     args = parser.parse_args()
 
     # Set image resolution
-    kwargs_call = {}
+    res = {}
```

Comment:
Why replace `kwargs_call` with `res` (this was old var name, may need to rebase w latest code)?

Reply:
I have rebased the patch. It is because I used infer_kwargs for saving the common parameters, Each pipeline only needs to add its own special parameters. The `res` is separated because sometimes there is no need to set it. Please see the lines:
```
    if args.distributed:
        with distributed_state.split_between_processes(args.prompts) as prompt:
            outputs = pipeline(prompt=prompt, **infer_kwargs, **res)
    else:
        outputs = pipeline(prompt=args.prompts, **infer_kwargs, **res)
```

Reply:
@yuanwu2017  Collecting all inference related keys/values per specific pipeline is OK strategy to make the codebase smaller.  You can leave `kwargs_call` to handle "inference kwargs".  And `res` keys/values can be included to same `kwargs_call` if there is a need to set it.  No need to separate these 2

Reply:
OK